client/js/nonprofits/donate/payment-step.js

```diff
@@ -170,10 +170,16 @@ const postTracking = (utmParams, donationResponse) => {
 }
 
 const postSuccess = (donationResponse) => {
-  // if (plausible) {
-  //   const resp = donationResponse()
-  //   plausible('payment_succeeded', {props: {amount: resp && resp.charge && resp.charge.amount && (resp.charge.amount / 100)}});
-  // }
+  try {
+    const plausible = window['plausible'];
```

Comment:
We weren't getting the `plausible` attribute from `window` and in some cases it didn't load. We get that attribute now which will be falsy if plausible wasn't loaded.